From 5279991f612e8dc57304dd6b5a3665deb4a6aa8a Mon Sep 17 00:00:00 2001
From: Bowgo Tsai <bowgotsai@google.com>
Date: Fri, 14 Sep 2018 12:49:14 +0800
Subject: [PATCH 17/26] Fix build break

Need to update prebuilts/api/28.0 to fix the build error caused by:

  https://android-review.googlesource.com/c/platform/system/sepolicy/+/741207

Bug: 113238007
Test: diff -rq system/sepolicy/prebuilts/api/28.0/private system/sepolicy/private
Change-Id: I83c3607f6a530f2fb842aaf39c123897b243f78f
---
 prebuilts/api/28.0/private/system_server.te | 9 ++++++++-
 1 file changed, 8 insertions(+), 1 deletion(-)

diff --git a/prebuilts/api/28.0/private/system_server.te b/prebuilts/api/28.0/private/system_server.te
index a1ea12f9..bbdd7ea5 100644
--- a/prebuilts/api/28.0/private/system_server.te
+++ b/prebuilts/api/28.0/private/system_server.te
@@ -455,7 +455,14 @@ allow system_server system_app_data_file:file create_file_perms;
 
 # Receive and use open app data files passed over binder IPC.
 # Types extracted from seapp_contexts type= fields.
-allow system_server { system_app_data_file bluetooth_data_file nfc_data_file radio_data_file shell_data_file app_data_file }:file { getattr read write append };
+allow system_server {
+  system_app_data_file
+  bluetooth_data_file
+  nfc_data_file
+  radio_data_file
+  shell_data_file
+  app_data_file
+}:file { getattr read write append map };
 
 # Access to /data/media for measuring disk usage.
 allow system_server media_rw_data_file:dir { search getattr open read };
-- 
2.17.1

